mlos_core/mlos_core/spaces/adapters/llamatune.py

```diff
@@ -139,6 +139,8 @@ def inverse_transform(self, configurations: pd.DataFrame) -> pd.DataFrame:
                     self.target_parameter_space,
                     vector=target_config_vector,
                 )
+                self.target_parameter_space.check_configuration(target_config)
```

Comment:
Might just need to clip them to the range in the case that it's too large (e.g., due to quantization issues).
Are there other failure cases than that?

Reply:
I took a slightly closer look at this.
`max_unique_values_per_param` is forcing the lower dimensions space to be `UniformIntegerHyperparatmers`, but the `vector` produced that is then dot producted with the approximate inverted matrix returns floats.

So, I think there's an additional scaling operation required to put that vector back into the `max_unique_values_per_param` range when that option `is not None`.

Reply:
Ok, I've added the q_scaler inverse transform here as well, which makes the values look much better, but one is still out of bounds.  I think it's maybe just needing clipping either there, or in a previous step.

Reply:
@kkanellis

Reply:
```sh
pytest -n0 mlos_bench/mlos_bench/tests/optimizers/llamatune_opt_test.py
```

to reproduce